### PR TITLE
ADM-596:[frontend]feat: Optimize the hierarchy of metrics page

### PR DESCRIPTION
**Pull request by LoisHuang2022**
## Summary

ADM-596:[frontend]feat: Optimize the hierarchy of metrics page

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: frontend/cypress/e2e/createANewProject.cy.ts [hunk 1]
```diff
@@ -246,7 +248,7 @@ describe('Create a new project', () => {
 
     nextButton().should('be.disabled')
 
-    cy.contains('Crews setting').should('exist')
+    cy.contains('Crews settings').should('exist')
```
Comment (guzhongren) [inline frontend/cypress/e2e/createANewProject.cy.ts:251]: `Crews` or `Crew`, just a question

**Review by guzhongren — APPROVED** [review]
Update with the comments

File: frontend/src/components/Common/MetricsSettingTitle/index.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import styled from '@emotion/styled'
-import { Divider } from '@src/components/Common/MetricsSettingTitle/style'
+
+export const MetricsSettingTitleContainer = styled.div({
+  margin: '1.25rem 0',
+  fontFamily: '"Roboto","Helvetica","Arial",sans-serif',
```
[the remark this answers is not in the document]
Comment (LoisHuang2022) [reply frontend/src/components/Common/MetricsSettingTitle/index.tsx:5]: Font-weight is added according to the design drawing given by the ux, maybe 'sans-serif' is enough? Actually I copy the '"Roboto","Helvetica","Arial",sans-serif' from the material UI component.

**Comment by guzhongren** [comment]
You can add `Roboto` and `Open Sans` for the body as a global font @LoisHuang2022

File: frontend/cypress/e2e/createANewProject.cy.ts (hunk 1 above)
Comment (LoisHuang2022) [reply frontend/cypress/e2e/createANewProject.cy.ts:251]: Crew

**Pull request merged**